index.html

```diff
@@ -4,29 +4,48 @@
     <link rel="preconnect" href="https://fonts.googleapis.com" />
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
 
-    <link rel="shortcut icon" href="/favicon.ico" />
+    <meta name="title" content="Bellshade - Learn with Our Community" />
+    <meta name="description" content="Komunitas untuk mengelola dan menulis kode untuk proyek pendidikan sumber terbuka Indonesia." />
+    <meta name="keywords" content="bellshade, komunitas, wpu, web programming unpas, community, programming, unpas" />
+    <meta name="robots" content="index, follow" />
+    <meta name="googlebot" content="index, follow" />
+    <meta name="msnbot" content="index, follow" />
+    <meta name="yahoobot" content="index, follow" />
+    <meta name="bingbot" content="index, follow" />
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <meta name="author" content="Bellshade" />
+    <meta name="copyright" content="Bellshade" />
+    <meta name="theme-color" content="#ffffff" />
+    <link rel="manifest" href="/manifest.webmanifest" />
+    <link rel="canonical" href="https://bellshade.github.io/" />
 
-    <title>Bellshade - Learn with Our Community</title>
+    <link rel="apple-touch-icon" href="/icons/icon-72x72.png" />
+    <link rel="apple-touch-icon" href="/icons/icon-96x96.png" />
+    <link rel="apple-touch-icon" href="/icons/icon-128x128.png" />
+    <link rel="apple-touch-icon" href="/icons/icon-144x144.png" />
+    <link rel="apple-touch-icon" href="/icons/icon-152x152.png" />
+    <link rel="apple-touch-icon" href="/icons/icon-192x192.png" />
+    <link rel="apple-touch-icon" href="/icons/icon-384x384.png" />
+    <link rel="apple-touch-icon" href="/icons/icon-512x512.png" />
+    <meta name="apple-mobile-web-app-status-bar" content="#ffffff" />
 
-    <meta name="description" content="Komunitas untuk mengelola dan menulis kode untuk proyek pendidikan sumber terbuka Indonesia" />
-    <meta property="og:locale" content="id_ID" />
+    <meta property="og:url" content="https://bellshade.github.io/" />
     <meta property="og:type" content="website" />
-    <meta property="og:title" content='Bellshade - Learn with Our Community' />
+    <meta property="og:title" content="Bellshade - Learn with Our Community" />
     <meta property="og:description" content="Komunitas untuk mengelola dan menulis kode untuk proyek pendidikan sumber terbuka Indonesia" />
-    <meta property="og:image" content="https://cdn.statically.io/img/media.discordapp.net/w=800/attachments/891222600422936616/892682527720996884/unknown.png" />
-    <meta property="og:image:width" content="800" />
-    <meta property="og:image:height" content="856" />
-    <meta name="twitter:card" content="summary_large_image" />
+    <meta property="og:image" content="https://cdn.statically.io/img/raw.githubusercontent.com/yuimatcha/bellshade.github.io/main/public/bellshade-logo.png" />
+
```

Comment:
Best practice dari facebook untuk open graph ditambahin ukuran gambar biar tau ukurannya tanpa mesti ngefetch gambarnya

```suggestion
    <meta property="og:image:width" content="1920" />
    <meta property="og:image:height" content="1080" />
```

Reply:
Mantavv koreksinya 👍🏻